File: OOP/Lab_1/src/Main.java
public class Main{
    public static void main(String[]args){
        lab_1 obj1 = new lab_1(0,0);
        lab_1 obj2 = new lab_1(0,0);
        obj1.setValue();
        obj1.getValue();
        obj2.setValue();
        System.out.println("Before:");
        obj2.getValue();
        obj2.sqr();
        System.out.println("After:");
        obj2.getValue();
        //array
        lab_1[] objArray;
        objArray = new lab_1[2];
        objArray[0]=obj1;
        objArray[1]=obj2;
        objArray[1].getValue();
        objArray[1].setValue();
        objArray[1].getValue();
        }
}

File: OOP/Lab_1/main.cpp
#include "lab_1.h"
int main(){
  lab_1 obj1;
  lab_1 obj2;
  lab_1 obj3;
  lab_1 obj4;
  //--Basic io example--
  cout<<"Testing obj1: "<<endl;
  obj1.setValue();
  obj1.getValue();
  //--sqr method example--
  cout<<"Testing obj2: "<<endl;
  obj2.setValue();
  cout<<"Before:"<<endl;
  obj2.getValue();
  obj2.sqr();
  cout<<"After:"<<endl;
  obj2.getValue();
  //--Array--
  cout<<"lab_1 array: "<<endl;
  array <lab_1, 2> objArray;
  for(int i=0;i<2;i++){
      objArray[i].setValue();
      objArray[i].getValue();
  }
<<<<<<< HEAD
  //--Composition|Assignment|Negative Value-- 
=======
  //--Composition|Assignment|Negative Value--
>>>>>>> da7f04177b9b647ed226dbd8e3599dc17a8a58e9
  cout<<"Array composition as obj3: "<<endl;
  obj3=objArray[0]*objArray[1];
  obj3.getValue();
  cout<<"obj4 as obj3 negative: "<<endl;
  obj4=-obj3;
  obj4.getValue();
  return 0;
}
